python/lsst/afw/geom/ellipses/axes.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from ._axes import *
+
+__all__ = []
+
+def __repr__(self):
+    return "Axes(a=%r, b=%r, theta=%r)" % (self.getA(), self.getB(), self.getTheta())
+
+def __reduce__(self):
+    return (Axes, (self.getA(), self.getB(), self.getTheta()))
+
+def __str__(self):
+    return "(a=%s, b=%s, theta=%s)" % (self.getA(), self.getB(), self.getTheta())
```

Comment:
picky but...please keep repr and str together (e.g. put reduce before or after both of them). That keeps related code together and makes it easier to see how str and repr differ.

Also, I suggest assigning the function right after declaring it. That keeps related code together.